tasks/deploy_apps.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
The name of this task set is confusing and does not reveal what it does. What about naming it `manage_apps.yml` or alike?

Reply:
I going to do this **after** including your PR.